python/huggingfaceserver/huggingfaceserver/vllm/vllm_completions.py

```diff
@@ -343,12 +357,15 @@ def apply_chat_template(
         self,
         messages: Iterable[ChatCompletionRequestMessage,],
     ):
-        return self.tokenizer.apply_chat_template(conversation=messages, tokenize=False)
+        return self.tokenizer.apply_chat_template(
+            conversation=messages, tokenize=False, add_generation_prompt=True
```

Comment:
Ideally this should be a request specific parameter with True as default. For that we need to modify the CreateChatCompletionRequest. [vLLM does that here](https://github.com/vllm-project/vllm/blob/7cd2ebb0251fd1fd0eec5c93dac674603a22eddd/vllm/entrypoints/openai/protocol.py#L177)

Reply:
I agree. We have a separate PR. Let's take this discussion to #3775